docs/modules/user-guide/partials/proc_adding-a-volume-component-to-a-devfile.adoc

```diff
@@ -0,0 +1,34 @@
+[id="proc_adding-a-volume-component-to-a-devfile_{context}"]
+= Adding a volume component to a devfile
+
+[role="_abstract"]
+You can use a volume component in order to share files among container components, making it easier to collaborate with other teams during the development process.
+
+.Prerequisites
+
+* xref:adding-schema-version-to-a-devfile.adoc[]
+* xref:adding-a-name-to-a-devfile.adoc[]
+* xref:adding-components-to-a-devfile.adoc[]
+
+.Procedure
+
+* Define a `volume` component in your devfile.
++
+.A minimal `volume` component
+[source,yaml]
+----
+schemaVersion: 2.1.0
+metadata:
+    name: mydevfile
+components:
+  - name: mydevfile
+    volume:
+      size: 2Gi
+----
++
+
+[role="_additional-resources"]
+.Additional resources
```

Comment:
This heading seems to be missing from the preview: https://devfile-docs-preview-pr-132.surge.sh/devfile/2.2.0/user-guide/adding-a-volume-component-to-a-devfile.html

Reply:
+1 to Rolfe's comment. @jc-berger PTAL and rectify.